src/test/java/teammates/test/cases/automated/FeedbackSessionClosingRemindersActionTest.java

```diff
@@ -94,6 +95,9 @@ public void allTests() throws Exception {
         session3.setSentOpenEmail(false); // fsLogic will set the flag to true
         verifyPresentInDatastore(session3);
 
+        // wait for very briefly so that the above session will be within the time limit
+        ThreadHelper.waitFor(5);
```

Comment:
I am a bit worrying about adding checks like this for some reasons:
- You don't know when you should add the operation (all read-after-write so some cases)
- The checks might be brittle

Might be we want to turn on the strong consistency settings for local testing.

Reply:
The check is not brittle, and this has nothing to do with weak/strong consistency. You only need to guarantee a one millisecond wait between when the session closing time is updated to be (now + 1 day) and when the action which queries the session is called. In some environments (especially AppVeyor) the time taken between the two blocks is apparently less than 1ms.